geowebcache/s3storage/src/main/java/org/geowebcache/s3/S3BlobStore.java

```diff
@@ -311,7 +311,7 @@ public KeyVersion apply(long[] loc) {
     @Override
     public boolean delete(final TileRange tileRange) throws StorageException {
 
-        final String coordsPrefix = keyBuilder.coordinatesPrefix(tileRange, false);
+        final String coordsPrefix = keyBuilder.coordinatesPrefix(tileRange, true);
```

Comment:
The change looks legit, not sure why it was working with the listener thought?
Also, can you add a test to cover this issue?

Reply:
Hi Andrea, thanks for your review.

It was working with the listener because the `coordsPrefix` is not used there (see lines 349 and on). 
`if (listeners.isEmpty()) {
<not working, because of coordsPrefix without ending slash>
} else { <working, not using coordsPrefix> }`

https://github.com/geowebcache/geowebcache/blob/a1a70d33facd19445e16abab6f12b35995b62d41/geowebcache/s3storage/src/main/java/org/geowebcache/s3/S3BlobStore.java#L349-L364

The test exists, but has been ignored by https://github.com/GeoWebCache/geowebcache/commit/855ed094b9269a3b919c34aac8e05aff05eec54c.
https://github.com/GeoWebCache/geowebcache/blob/b4dff9c2244fe4569319fe82b453f4f82387fd5f/geowebcache/s3storage/src/test/java/org/geowebcache/s3/AbstractS3BlobStoreIntegrationTest.java#L427

It effectively fails before the change above, and succeeds after the change.

Maybe you want to try to un-ignore this test?

Reply:
Done.